sentinel-core/pom.xml

```diff
@@ -28,6 +28,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-all</artifactId>
```

Comment:
No extra dependencies should be introduced in `sentinel-core`.

Reply:
I will modify it in the following pr.